internal/website/go.mod

```diff
@@ -0,0 +1,8 @@
+module gocloud.dev/internal/website
```

Comment:
My thought was in case we end up creating more scripts like this to deal with the website.

My pragmatic reason was that the VSCode workspace I checked in already contains the website as a top-level folder (and #1883 documents that the workspace must be 1:1 with module roots), and I didn't want to create another entry. 😄